tests/migration/online_migration/yast2_migration.pm

```diff
@@ -150,7 +150,9 @@ sub run {
         zypper_call "rs $service{$addon}\t";    # remove service
     }
 
-    script_run("yast2 migration; echo yast2-migration-done-\$? > /dev/$serialdev", 0);
+    my $migration_cmd = 'migration';
+    $migration_cmd = 'migration_sle' if (get_var('LEAP_TECH_PREVIEW'));
```

Comment:
Suggest change to:
$migration_cmd = get_var('LEAP_TECH_PREVIEW') ? 'migration_sle' : 'migration';

Reply:
changed && thanks